CHANGES.rst

```diff
@@ -577,6 +577,11 @@ ramp_fitting
   removing NaNs from the rateints product and setting appropriate DQ
   flags. [#6949]
 
+refpix
+------
+
+- Set DQ of ref pixels to DO_NOT_USE after `refpix` correction [#7017]
+
```

Comment:
```suggestion
```

now that #8671 is merged (switching change log handling to `towncrier`) this change log entry should be a file in `changes/` instead:
```shell
echo "Set DQ of ref pixels to ``DO_NOT_USE`` after ``refpix`` correction" > changes/7017.refpix.rst
```

(new PRs will include instructions on how to do this)

Reply:
(ignore this if this PR is no longer active or needed)